libafl/src/bolts/llmp.rs

```diff
@@ -255,15 +255,15 @@ unsafe fn _llmp_page_init<SHM: ShMem>(shmem: &mut SHM, sender: u32, allow_reinit
     };
     (*page).magic = PAGE_INITIALIZED_MAGIC;
     (*page).sender = sender;
-    ptr::write_volatile(&mut (*page).current_msg_id, 0);
+    ptr::write_volatile(ptr::addr_of_mut!((*page).current_msg_id), 0);
```

Comment:
That's a great catch! Everything should be nice and aligned, but this seems better. 
Where did the warnings come up btw?

Reply:
I use nightly mainly

Reply:
So stable may get these warnings eventually, interesting